test/functionalTest/cases/0000_ngsi10_convenience/subscriptions_onchange.test

```diff
@@ -29,7 +29,6 @@ brokerStart CB
 accumulatorStart
```

Comment:
We have decided to defer this to a short future (see https://github.com/telefonicaid/fiware-orion/issues/1476#issuecomment-159562257). However at least a subscriptions_onchange.test.DISABLED with the original content should be created in this PR, to ease the future work (otherwise, we would need to do some archeology work in develop branch).